This patch allows the program to run on platforms, where
pointers are wider than 32-bit...

	-mi

--- src/BOB_killer.cc	2005-03-01 09:15:08.000000000 -0500
+++ src/BOB_killer.cc	2007-07-27 13:49:30.000000000 -0400
@@ -271,10 +271,10 @@
 
 	// tables of offsets and counters (words and bytes)
-	BOBtableP1 = (Sint16 **)(memGestion->reserveMem(sizeof(Sint32) * animationN, 0x424F4250));
+	BOBtableP1 = (Sint16 **)(memGestion->reserveMem(sizeof(*BOBtableP1) * animationN, 0x424F4250));
 	error_init(memGestion->retour_err());
 	if(erreur_num) return erreur_num;
 
 	// tables of data (pixels of the sprite)
-	BOBtableP2 = (char **)(memGestion->reserveMem(sizeof(Sint32) * animationN, 0x424F4250));
+	BOBtableP2 = (char **)(memGestion->reserveMem(sizeof(*BOBtableP2) * animationN, 0x424F4250));
 	error_init(memGestion->retour_err());
 	if(erreur_num) return erreur_num;
@@ -282,5 +282,5 @@
 	// tables of offsets and counters (byte peer byte)
 	if(fTableByte) 
-	{	BOBtableP3 = (Sint16 **)(memGestion->reserveMem(sizeof(Sint32) * animationN, 0x424F4250));
+	{	BOBtableP3 = (Sint16 **)(memGestion->reserveMem(sizeof(*BOBtableP3) * animationN, 0x424F4250));
 		error_init(memGestion->retour_err());
 		if(erreur_num) return erreur_num;
@@ -358,5 +358,5 @@
 	//###################################################################
 	adresseTAB = (char **)
-		(memGestion->reserveMem(sizeof(Sint32) * animationN, 0x424F4250));
+		(memGestion->reserveMem(sizeof(*adresseTAB) * animationN, 0x424F4250));
 	error_init(memGestion->retour_err());
 	if(erreur_num) return erreur_num;
